rocthrust/PKGBUILD

```diff
@@ -1,50 +1,36 @@
 # Maintainer: Markus Näther <naetherm@informatik.uni-freiburg.de>
 pkgname=rocthrust
 pkgver=3.3.0
-pkgrel=3
-pkgdesc="Port of the Thrust parallel algorithm library atop HIP/ROCm."
+pkgrel=4
+pkgdesc="Port of the Thrust parallel algorithm library atop HIP/ROCm"
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocThrust"
```

Comment:
Can we change the url to https://rocm-documentation.readthedocs.io/en/latest/ROCm_Libraries/ROCm_Libraries.html#rocthrust

Reply:
Sure. But shouldn't we change this for all packages in this repo then? At the moment they all point to the github repos. As far as I can see, the documentation of ROCm is just the README.md from the repos.

Reply:
We should but for now lets just change them one at a time as they come up since it is not a critical issue.

Reply:
I've changed the url.